packages/message/src/Collections.ts

```diff
@@ -25,6 +29,53 @@ import { Message } from '@cumulus/types';
 export const constructCollectionId = (name: string, version: string) =>
   `${name}___${version}`;
 
+/**
+ * Get collection name from execution message.
+ *
+ * @param {Message.CumulusMessage} message - An execution message
+ * @returns {string | undefined} - Collection name or undefined
+ *
+ * @alias module:Collections
+ */
+export const getCollectionNameFromMessage = (
```

Comment:
Nit: Is there a reason we're exporting these versus just exposing getCollectionInfoFromMessage?

I'm wondering if exposing the name/version methods and their units could be eliminated here for complexity/interface tidiness reasons.